package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nemo",
-  "version": "4.2.2",
+  "version": "4.2.3",
```

Comment:
confused about this change in the lockfile. presumably an artifact of having earlier revved the patch version? i'll make sure this reads correctly before publishing.

Reply:
yeah, forgot about that, sorry